curriculum/challenges/english/25-front-end-development/workshop-merge-sort/656627b47bd2d2a4afbc945d.md

```diff
@@ -0,0 +1,43 @@
+---
+id: 656627b47bd2d2a4afbc945d
+title: Step 4
+challengeType: 20
+dashedName: step-4
+---
+
+# --description--
+
+In the previous step, you got the mid point. You can use it to divide `array` into two and assign each part to new variables. 
+
+Use the slice syntax to extract the left half of `array` and assign it to a variable named `left_part`.
+
+# --hints--
+
+You should have a variable named `left_part` in your `merge_sort` function.
+
+```js
+({
+  test: () => assert(runPython(`_Node(_code).find_function("merge_sort").has_variable("left_part")`)) 
+})
+```
+
+You should assign `array[:middle_point]` to the `left_part` variable.
```

Comment:
I tried writing this and was marked wrong.

```py
left_part=array[0:middle_point]
```

if we are looking specifically for `array[:middle_point]`, then that should be explicitly mentioned in the instructions. or the tests should be updated to accept both.